applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -101,6 +101,10 @@ For information about how to upgrade the device firmware using a PC or a smartph
 .. note::
     Currently the DFU over Bluetooth LE SMP and Matter OTA are not available for the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
 
+There is an additional functionality, which allows the Matter bridge device to work as a smart plug device.
```

Comment:
I would add one more sentence after that. I think we could inform that in this scenario, the Matter bridge exposes additional endpoint with id ``2`` that represents Matter on/off smart plug device type functionality.